components/devtools/network_handler.rs

```diff
@@ -26,30 +26,29 @@ pub(crate) fn handle_network_event(
     netevent_actor_name: String,
     mut connections: Vec<TcpStream>,
     network_event: NetworkEvent,
-    browsing_context_actor_name: String,
+    watcher_name: String,
```

Comment:
We don't need to pass this here; it's already stored as a field in NetworkEventActor.